couchbase/src/main/scala/akka/stream/alpakka/couchbase/javadsl/CouchbaseSource.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.couchbase.javadsl
+
+import akka.NotUsed
+import akka.stream.javadsl.Source
+import com.couchbase.client.java.Bucket
+import com.couchbase.client.java.document.Document
+import com.couchbase.client.java.query.{AsyncN1qlQueryRow, N1qlQuery, Statement}
+import scala.collection.JavaConverters._
+import scala.collection.immutable.Seq
+
+object CouchbaseSource {
+
+  def fromN1qlQuery(query: N1qlQuery, couchbaseBucket: Bucket): Source[AsyncN1qlQueryRow, NotUsed] =
+    Source.fromGraph(akka.stream.alpakka.couchbase.scaladsl.CouchbaseSource.fromN1qlQuery(query, couchbaseBucket))
```

Comment:
Equivalent to `akka.stream.alpakka.couchbase.scaladsl.CouchbaseSource.fromN1qlQuery(query, couchbaseBucket).asJava`

Reply:
Thanks a lot :)